zfs_snapshot.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Make sure you have set -e at the start of the script, as otherwise any error in the script does not cause it to stop